src/jit/codegenlegacy.cpp

```diff
@@ -11715,10 +11742,9 @@ void                CodeGen::genCodeForTreeSmpOpAsg(GenTreePtr tree)
 
         regGC = WriteBarrier(op1, op2, addrReg);
 
+        // No, assignment was not done by the WriteBarrier
```

Comment:
This comment really belongs below the next line - or could be reworded as "Was assignment done by the WriteBarrier?"